<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokerServeurGui</class>
 <widget class="QMainWindow" name="PokerServeurGui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Poker - Serveur</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="logs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSend">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ExÃ©cuter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
